cmake/partition_manager.cmake

```diff
@@ -47,19 +59,62 @@ if(PM_IMAGES OR (EXISTS ${static_configuration_file}))
   # Add subsys defined pm.yml to the input_files
   list(APPEND input_files ${PM_SUBSYS_PREPROCESSED})
 
+  math(EXPR ram_size "${CONFIG_SRAM_SIZE} * 1024" OUTPUT_FORMAT HEXADECIMAL)
```

Comment:
Is this being used anywhere? Same with flash can't see any reference to it.

Reply:
You are correct this belongs in a different PR :)